src/descriptor/mod.rs

```diff
@@ -773,6 +890,11 @@ where
                         }
                     }
                     _ => {
+                        if top.args[0].name == "sortedmulti" {
```

Comment:
I would rather you moved this up a level, and match on `("sortedmulti", _)`, rather than adding an if-check to the `_` branch.

Reply:
Done